flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1456,4 +1456,7 @@
     "org.srb2.SRB2Kart": {
         "flathub-json-modified-publish-delay": "extra-data"
     }
+    "io.github.hakandundar34coding.system-monitoring-center": {
+        "finish-args-flatpak-spawn-access": "the app predates this linter rule"
```

Comment:
I mean, that's not true? Can you put actual use-case here?

Reply:
Updated the file again. A lot of applications used the previous explanation.

Reply:
because we had no explanation and we added the linter rules. This app does't predate the rule.